k8s/vizier/bootstrap/cert_provisioner_role.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pl-cert-provisioner-service-account
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: pl-cert-provisioner-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: pl-cert-provisioner-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: pl-cert-provisioner-role
+subjects:
+- kind: ServiceAccount
+  name: pl-cert-provisioner-service-account
+  namespace: pl
```

Comment:
We support Pixie deploys in namespaces other than pl. How we do that is that we actually generate templates from these base kustomize YAMLs using: https://github.com/pixie-io/pixie/blob/main/src/utils/template_generator/vizier_yamls/vizier_yamls.go

You will need to add something like this: https://github.com/pixie-io/pixie/blob/main/src/utils/template_generator/vizier_yamls/vizier_yamls.go#L404-L413
for this new rolebinding.

Reply:
Fixed on latest!